mw-webapp/src/GlobalContext.ts

```diff
@@ -8,6 +8,13 @@ const DEFAULT_SET_USER = () => {
   throw Error("The user context is not initialized properly");
 };
 
+/**
+ * Default setIsInitialization
+ */
+const DEFAULT_SET_INITIALIZATION = () => {
+  throw Error("The app is not initialization");
```

Comment:
Let's s use more specific message, like "This is stub function for the initialization. This function should not be called"

Reply:
fixed